### PR TITLE
counsel.el : Add counsel-separator face

### PR DESCRIPTION
**Pull request by tttuuu888**
First of all, thanks for this great package.
I added counsel-separator face to make it easy to distinguish texts on counsel-yank-pop list.
when I eval `(setq counsel-yank-pop-separator "\n----------\n")`, separators on counsel-yank-pop list will be looked as below.

s1

s2

Thanks.

### CODE REVIEW

File: counsel.el [hunk 1]
```diff
@@ -730,6 +730,12 @@ a symbol and how to search for them."
   "Face used by `counsel-M-x' for key bindings."
   :group 'ivy-faces)
 
+;;** `counsel-yank-pop'
```
Comment (basil-conto) [inline counsel.el]: This heading already exists later on in the file, no need to have it twice.

File: counsel.el [hunk 2]
```diff
@@ -730,6 +730,12 @@ a symbol and how to search for them."
...
+(defface counsel-separator
```
Comment (basil-conto) [inline counsel.el]: The docstring sounds pretty generic, perhaps this face should be added straight to `ivy.el`?

**Comment by tttuuu888** [comment]
@basil-conto Thanks for your review. I just added a commit to rename 'counsel-separator' to 'ivy-separator'.

**Comment by basil-conto** [comment]
Thanks @tttuuu888, LGTM.

**Comment by abo-abo** [comment]
Merged, thanks.

**Pull request closed without merging**